.github/workflows/github-ci.yml

```diff
@@ -45,6 +46,13 @@ jobs:
       - name: Run screenshot tests
         run: ./gradlew designsystem:verifyPaparazziDebug
 
+      - name: Upload Paparazzi report
```

Comment:
I'm not sure if this is adding value, as if the verification fails, we need to work on the branch so that we can check the reports locally